README.md

```diff
@@ -28,10 +28,10 @@
 
 <!-- PROJECT DESCRIPTION -->
 
```

Comment:
- [ ] Kindly remove these files as they would not be needed and we want to keep our codebase clean.

  ![oas](https://user-images.githubusercontent.com/81468675/219040453-7ba5ea9a-ab59-4c1d-8f77-30758f0e6518.jpg)